### PR TITLE
Hifi/bc 640 hifi sign up and log in

### PR DESCRIPTION
**Pull request by GeoSauer**
- Adds assets and finalizes styling of `sign-up`, `log-in`, and `confirmation-email-sent` pages.
- Fixes a nested `<form>` error caused by password inputs.
- Tweaks color/rendering of PasswordCriteria to match Figma HiFi.

 [Jira ticket](https://bootcamper.atlassian.net/browse/BC-640)

### CODE REVIEW

File: src/screens/Auth/SignUp/EmailSentConfirmation/EmailSentConfirmation.scss [hunk 1]
```diff
@@ -4,43 +4,60 @@
...
 
     &.text {
       font-size: 16px;
       font-weight: 400;
-      margin-bottom: 50px;
-      width: 463px;
+      line-height: 24px;
+      margin-bottom: 32px;
+      width: 456px;
+      text-align: left;
```
Comment (gooliajulia) [inline src/screens/Auth/SignUp/EmailSentConfirmation/EmailSentConfirmation.scss:34]: alphabetize please 🤓

File: src/screens/Auth/SignUp/EmailSentConfirmation/EmailSentConfirmation.tsx [hunk 2]
```diff
@@ -42,14 +43,19 @@ export const EmailSentConfirmation: React.FC = () => {
...
         </p>
-        <p>The link provided in the email will expire in 30 minutes. </p>
       </div>
       <button className='resend-link' onClick={handleResendEmailClick}>
         Re-send email
       </button>
+      <div className='img-container'>
+        <img
+          src={emailSentImage}
+          alt='A person jumps in the air in celebration'
```
Comment (gooliajulia) [inline src/screens/Auth/SignUp/EmailSentConfirmation/EmailSentConfirmation.tsx:56]: nice alt use 👏

**Review by gooliajulia — CHANGES_REQUESTED** [review]
Excellent work 👏 Props for the attention to detail and implementation on the Sign up screen password validation text depending on user's input focus 🔥 

Besides the other comments on PR a couple small nits to match the Figma:
* sample email address in parenthesis on sign up screen is different from figma
* The Sign up font in the submit button should be a shade of blue when disabled and another shade of blue when enabled to match Figma (currently black it looks like)

File: src/components/Inputs/PasswordInputs/ConfirmNewPassword.tsx [hunk 3]
```diff
@@ -43,17 +52,32 @@ export const ConfirmNewPassword = ({
...
+            }}
           />
-          <IconButton
-            className='confirm-password eyecon'
-            aria-label='toggle password visibility'
-            onClick={() => toggleVisiblity(inputType, setInputType)}
-          >
-            {inputType === 'password' ? <VisibilityOff /> : <Visibility />}
-          </IconButton>
+          {isValid === false ? (
```
[the remark this answers is not in the document]
Comment (GeoSauer) [reply src/components/Inputs/PasswordInputs/ConfirmNewPassword.tsx:59]: Ah yea I think I had just context switched from using a variable that could be `true`, `false, or `null`. Good catch!

**Review by gooliajulia — APPROVED** [review]
awesome. thanks for the quick addressing of comments. lgtm!

**Pull request merged**